server/web/public.js

```diff
@@ -15,6 +15,23 @@ exports.register = function(server, options, next) {
     }
   });
 
+  server.route({
+    method: "GET",
+    path: "/robots.txt",
+    handler: {
+      file: "public/robots.txt"
+    }
+  });
+
+  // TODO: should this be /public/ like non-precomposed?
```

Comment:
`precomposed` nowadays isn’t really needed anymore (was only for old iOS). Let’s just do `/apple-touch-icon.png`.